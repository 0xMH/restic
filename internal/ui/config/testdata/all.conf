repo = "sftp:user@server:/srv/repo"
password = "secret"
password_file = "/root/secret.txt"

backup {
    target = [
        "/home/user/",
        "/home/otheruser",
    ]

    exclude = ["*.c"]
}

backend "local" {
    type = "local"
    path = "/foo/bar"
}

backend "local2" {
    path = "/foo/bar"
}

backend "sftp" {
    type = "sftp"
    user = "foo"
    host = "bar"
    path = "/foo/bar"
}
