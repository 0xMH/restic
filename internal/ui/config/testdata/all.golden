{
  "Repo": "sftp:user@server:/srv/repo",
  "Password": "secret",
  "PasswordFile": "/root/secret.txt",
  "Backends": {},
  "Backup": {
    "Target": [
      "/home/user/",
      "/home/otheruser"
    ],
    "Excludes": [
      "*.c"
    ]
  }
}
